packages/provider/src/RelayProvider.ts

```diff
@@ -131,7 +141,33 @@ export class RelayProvider implements HttpProvider, Web3ProviderBaseInterface {
     })
   }
 
-  _ethSendTransaction (payload: JsonRpcPayload, callback: JsonRpcCallback): void {
+  /**
+   * The ID can be either a RelayRequestID which requires event-based lookup or Transaction Hash that goes through
+   * @param payload
+   * @param callback
+   */
+  async _ethGetTransactionReceipt (payload: JsonRpcPayload, callback: JsonRpcCallback): Promise<void> {
```

Comment:
ugly merge. I think all internal funcs should be async and not callback (there are good "promisify" (and vice-versa) tools
this way, you can also easily call in parallel the getByHash and getByRequestId